src/core/components/AuthenticateButton/index.js

```diff
@@ -53,9 +55,17 @@ export const mapStateToProps = (state) => ({
 });
 
 export const mapDispatchToProps = (dispatch) => ({
-  handleLogOut() {
+  handleLogOut({ api }) {
+    const { location } = window;
+    if (config.get('apiHost') === `${location.protocol}//${location.host}`) {
```

Comment:
I'm concerned with including the if branch for this code because it seems fragile. I think letting the logout feature break for non-proxy developers is fine. We will tell everyone to use the proxy and if we run into problems with it then we could add the if branch back in.